apps/server/test/lexical/server/code_intelligence/entity_test.exs

```diff
@@ -299,30 +304,217 @@ defmodule Lexical.Server.CodeIntelligence.DefinitionTest do
     test "find the definition when calling a erlang module", %{project: project} do
       subject_module = ~q[
         :erlang.binary_to_atom|("1")
-      ]
+        ]
       {:ok, uri, definition_line} = definition(project, subject_module)
 
       assert uri =~ "/src/erlang.erl"
       assert definition_line =~ ~S[«binary_to_atom»(Binary)]
     end
   end
 
-  defp caller_position(subject_module) do
-    {line, character} = cursor_position(subject_module)
-    Position.new(line, character)
+  describe "resolve/2" do
+    test "syntax error", %{project: project} do
+      code = ~q[
+        Some.Modul|e.
+        ]
+
+      assert {:error, _} = resolve(project, code)
+    end
+
+    test "returns an error if the cursor is not over an entity", %{project: project} do
+      code = ~q[
+        Beyond.The.End|
```

Comment:
I feel like this should return `Beyond.The.End`. The cursor is right next to a module name. I would expect the `{:error, :not_found}` to happen if there was a space between the end of the `End` module and the cursor.